_data/simple-icons.json

```diff
@@ -8934,6 +8934,11 @@
                 "type": "CC-BY-SA-4.0"
             }
         },
+        {
+            "title": "RustDocs",
```

Comment:
I don't see them refer to it as `RustDocs` anywhere, only `Docs.rs`. Maybe we should change it to that?

Reply:
Sure, I've changed it.